app/tailwind.config.js

```diff
@@ -51,19 +50,196 @@ module.exports = {
+                    500: '#00715e',
+                    600: '#008b74',
+                    700: '#00deba',
+                    800: '#00a68b',
+                    900: '#00c1a2',
+                    1000: '#cbfff6',
+                },
+                yellow: {
+                    100: '#211a00',
+                    200: '#312600',
+                    300: '#493900',
+                    400: '#644e00',
+                    500: '#7f6300',
+                    600: '#9b7900',
+                    700: '#f6c100',
+                    800: '#b99000',
+                    900: '#d7a800',
+                    1000: '#fff0bc',
+                },
+            },
+
+            /*--- TYPOGRAPHY –--*/
+            fontSize: { //to be aliased to be more legible and in-line with Tailwind Styles
+                title1: ['2.25rem', {
+                    lineHeight: 'normal',
+                    fontWeight: 'normal',
+                }],
+                title2: ['1rem', {
+                    lineHeight: 'normal',
+                    fontWeight: 'normal',
+                }],
+                title3: ['1rem', {
+                    lineHeight: 'normal',
+                    fontWeight: 'normal',
+                }],
+                largePlus: ['1rem', {
+                    lineHeight: '8.75',
+                    fontWeight: '500',
+                }],
+                large: ['1rem', {
+                    lineHeight: '8.75',
+                    fontWeight: 'normal',
+                }],
+                regularPlus: ['0.9375rem', {
+                    lineHeight: '8.75',
+                    fontWeight: '500',
+                }],
+                regular: ['0.9375rem', {
+                    lineHeight: '8.75',
+                    fontWeight: 'normal',
+                }],
+                smallPlus: ['0.8125rem', {
+                    lineHeight: '8.75',
+                    fontWeight: '500',
+                }],
+                small: ['0.8125rem', {
+                    lineHeight: '8.75',
+                    fontWeight: 'normal',
+                }],
+                miniPlus: ['1rem', {
+                    lineHeight: 'normal',
+                    fontWeight: '500',
+                }],
+                mini: ['1rem', {
+                    lineHeight: 'normal',
+                    fontWeight: 'normal',
+                }],
+                microPlus: ['0.6875rem', {
+                    lineHeight: 'normal',
+                    fontWeight: '500',
+                }],
+                micro: ['0.6875rem', {
+                    lineHeight: 'normal',
+                    fontWeight: 'normal',
+                }],
             },
-            borderRadius: {
+
+            /*--- NUMERIC VALUES –--*/
+            borderRadius: { //to be consolidated
                 lg: 'var(--radius)',
                 md: 'calc(var(--radius) - 2px)',
                 sm: 'calc(var(--radius) - 4px)',
+                'x-small': '2px',
```

Comment:
Actually I think I'll probably remove these radii – seems a bit over-defined for now.

Reply:
Less is small afterall unless you think it's going to be used a lot
`xs` is the convention for extra small

Reply:
I just removed it – Tailwind's border radii are pretty solid